hs_core/hydroshare/hs_bagit.py

```diff
@@ -119,13 +116,18 @@ def get_remote_file_manifest(tmpdir, resource):
             last_sep_pos = irods_file_name.rfind('/')
             ref_file_name = irods_file_name[last_sep_pos+1:]
             fetch_url = '{0}/django_irods/download/{1}'.format(utils.current_site_url(), resource.short_id + irods_file_name)
-            checksum = istorage.get_checksum(srcfile)
         else:
             irods_file_name = f.storage_path
             irods_dest_prefix = "/" + settings.IRODS_ZONE + "/home/" + settings.IRODS_USERNAME
             srcfile = os.path.join(irods_dest_prefix, irods_file_name)
             fetch_url = '{0}/django_irods/download/{1}'.format(utils.current_site_url(), irods_file_name)
+
+        checksum = None
+
+        try:
             checksum = istorage.checksum(srcfile)
+        except SessionException as ex:
+            raise HsBagitException(ex.message)
```

Comment:
@ismagilani replace ```ex.message``` with ```ex.stderr``` to capture the error message raised from SessionException